src/device/pf_port.c

```diff
@@ -225,15 +225,23 @@ pf_mediatype_values_t pf_port_get_media_type (pnal_eth_mau_t mau_type)
    case PNAL_ETH_MAU_RADIO:
       return PF_PD_MEDIATYPE_RADIO;
    case PNAL_ETH_MAU_COPPER_10BaseT:
+   case PNAL_ETH_MAU_COPPER_10BaseTX_HALF_DUPLEX:
+   case PNAL_ETH_MAU_COPPER_10BaseTX_FULL_DUPLEX:
```

Comment:
See comment below, suggestion to rename my badly named enums 10BaseTX -> 10BaseT.